examples/xml-io/xml_io.out

```diff
@@ -1,3 +1,5 @@
+# In the directory, which contains the `.bal` file, create a directory named `files`,
+# and add an XML file named `xmlFile.xml` in it.
```

Comment:
Creating the directory called `files` is enough. No need to put the XML there, it will be automatically created.

Reply:
Fixed.